src/components/new-request/stats.vue

```diff
@@ -123,6 +125,8 @@ export default class Stats extends Vue {
   padding-left: 8px;
   font-size: 12px;
   line-height: 18px;
+  text-decoration-line: underline;
+  text-decoration-style: dotted;
```

Comment:
Use base class `dotted-underline` instead.

Also, should this use class `cursor-default`?

Reply:
Not sure about your comment regarding `cursor-default`.  There is no mention of what the cursor needs to be in the designs from what I can see.

Reply:
I think we don't handle the tooltip hover cursor consistently in our apps...
Anyway, in this app, we don't change the cursor for tooltips, so this is fine.